doc/Building.md

```diff
@@ -20,7 +20,7 @@ functionalities such as rendering, serialization, file handling and network
 communication. Those dependencies are:
 * Boost 1.54 or later
 * MPI with MPI_THREAD_MULTIPLE support (openmpi 1.6.5 or later recommended)
-* Qt 5.4 or 5.5 (versions 5.6 and 5.7 currently have some
+* Qt 5.4, 5.5 or 5.7 (version 5.6 currently has some
```

Comment:
so 5.7 doesn't have the issues that 5.6 has? we could skip it then and tell people to use 5.7.